### PR TITLE
fix: update `Input` component

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

Remove the `padding` `styled` and move it to `className`, allowing it to be replaced.

**Before:**

Screenshot from 2020-07-13 18-33-31
Screenshot from 2020-07-13 18-33-12
Screenshot from 2020-07-13 18-32-54

**After:**

Screenshot from 2020-07-13 22-03-38
Screenshot from 2020-07-13 22-03-30
Screenshot from 2020-07-13 22-03-22

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/app/components/Input/InputGroup.tsx [hunk 1]
```diff
@@ -6,12 +6,12 @@ const InputAddon = styled.div`
 	${tw`absolute inset-y-0 flex items-center justify-center overflow-hidden`};
 `;
 
-export const InputAddonStart = styled(InputAddon)`
-	${tw`left-0 rounded-l border border-transparent`}
+export const InputAddonStart = styled.div`
+	${tw`border border-r-0 rounded-l`}
```
Comment (luciorubeens) [inline src/app/components/Input/InputGroup.tsx:10]: `InputAddonStart` and `InputAddonEnd` have the same purpose, it is just to position it absolute within a relative parent

File: src/app/components/Input/Input.tsx [hunk 2]
```diff
@@ -6,10 +6,10 @@ import { useFormField } from "../Form/useFormField";
 type InputProps = { as?: React.ElementType; isInvalid?: boolean } & React.HTMLProps<any>;
 
 const InputStyled = styled.input`
-	${tw`py-3 px-4`}
+	${tw`px-4 py-3`}
```
Comment (luciorubeens) [inline src/app/components/Input/Input.tsx:9]: It was broken by this line, having the padding in the `styled` does not allow it to be override, should be moved to `className`

Comment (brenopolanski) [reply src/app/components/Input/Input.tsx:9]: Yeah, make sense! Updating... :oleglala:

**Comment by faustbrian** [comment]
@brenopolanski conflicts

**Comment by brenopolanski** [comment]
done @faustbrian

**Pull request merged**